<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RunWidget</class>
 <widget class="QWidget" name="RunWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="5">
    <widget class="QLabel" name="description">
     <property name="text">
      <string>Notes on the simulation will be stored in the simulation folder upon running the simulation</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="noteBoxLabel">
     <property name="text">
      <string>Filename</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="5">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="5" column="5">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="6">
    <widget class="QTextEdit" name="noteBox">
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="4" colspan="2">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Save Simulation Settings</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QPushButton" name="runButton">
     <property name="text">
      <string>Run Simulations and Save</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="noteBoxName">
     <property name="text">
      <string>notes.txt</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
